aws/table_aws_elasticache_update_action.go

```diff
@@ -0,0 +1,174 @@
+package aws
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/service/elasticache"
+	"github.com/aws/aws-sdk-go-v2/service/elasticache/types"
+	elasticachev1 "github.com/aws/aws-sdk-go/service/elasticache"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsElastiCacheUpdateAction(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_elasticache_update_action",
+		Description: "AWS ElastiCache Update Action",
+		List: &plugin.ListConfig{
+			KeyColumns: plugin.OptionalColumns([]string{"cache_cluster_id", "replication_group_id"}),
+			Hydrate:    listElastiCacheUpdateActions,
```

Comment:
Please include the missing tags. Ref: https://github.com/turbot/steampipe-plugin-aws/pull/2439/files#diff-caef6a3568166825f778964b24715fb2aab41df81a997bc6a6745ca3767b45faR21